src/getHandlers.js

```diff
@@ -0,0 +1,42 @@
+import lifecycle from './lifecycle'
+
+function format (string) {
+  /*eslint-disable no-unused-vars*/
+  const [ _, action, state = 'done' ] = `${ string }`.match(/(\w*)\_\d+[\_\w]*/) || []
+  /*eslint-enable no-unused-vars*/
```

Comment:
Beyond that though, I want to say there's a thing you can do with `no-unused-vars` that ignores variables that start with `_`.